### PR TITLE
i18n(es): Fix typos `getting-started.mdx`, `project-structure.mdx`, `routing.mdx` & `configuring-astro.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- New or updated content
- Translated content
- Changes to the docs site code
- Something else!

#### Description

- What does this PR change? Give us a brief description.

Fix typos `getting-started.mdx`, `project-structure.mdx`, `routing.mdx` & `configuring-astro.mdx`

### I am participating in the Hacktoberfest!

### CODE REVIEW

File: src/content/docs/es/core-concepts/routing.mdx [hunk 1]
```diff
@@ -189,18 +189,18 @@ Debido a que las páginas SSR no pueden usar `getStaticPaths`, no pueden recibir
...
-		title: 'Astro products',
-		text: 'We have lots of products for you',
+		title: 'Productos de Astro',
+		text: 'Tenemos muchos productos para ti',
 	},
 	{
 		slug: 'products/astro-handbook',
-		title: 'The ultimate Astro handbook',
-		text: 'If you want to learn Astro, you must read this book.',
+		title: 'El libro definitivo de Astr',
```
Comment (santiemanuel) [inline src/content/docs/es/core-concepts/routing.mdx:202]: Typo. Faltó la o en Astro.

**Review by dreyfus92 — APPROVED** [review]
Solo tendrías que agregar el cambio sugerido por @santiemanuel. Aparte de eso LGTM ✅

File: src/content/docs/es/core-concepts/routing.mdx (hunk 1 above)
Comment (Waxer59) [inline src/content/docs/es/core-concepts/routing.mdx:202]:
```suggestion
		title: 'El libro definitivo de Astro',
```

**Comment by Waxer59** [comment]
Gracias por la review @dreyfus92 y @santiemanuel 🤠!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**